workers.tf

```diff
@@ -122,3 +122,8 @@ resource "null_resource" "tags_as_list_of_maps" {
     "propagate_at_launch", "true"
   )}"
 }
+
+resource "aws_iam_service_linked_role" "elasticloadbalancing" {
```

Comment:
Should this be inside `workers.tf`? This role is used by the cluster.

Reply:
Good point. I'll move it.